### ISSUE TITLE
Training questions

### ISSUE THREAD

**Issue by nom** [alimama-creative/FLUX-Controlnet-Inpainting#5]
Hi, great work on the first working inpainting model for Flux!

I'm working on reproducing the training to make sure I can finetune the model. Right now, my finetuning seems to destroy the inpainting after only 100 steps of training. I'm wondering what it could be. 
1. One key hyperparameter seems to be the controlnet conditioning_scale. What did you set it to for training? The default is 1.0, the docs recommend a range of 0.9-0.95 for inference and the docstring example says 0.6.
2. Which parameters did you train and which did you freeze?
3. Any other gotchas for training?

Thanks!

**Comment by JPlin** [comment]
conditioning_scale is set to 1 during training, and the docstring is incorrect; please refer to main.py. The transformer has been fixed, and the ControlNet part has been trained；Fine-tuning the distilled model without distillation loss can be quite unstable, so it's necessary to check different checkpoints frequently.

**Comment by nom** [comment]
Thanks @JPlin. Curious how many steps you trained for and at what batch size?

**Comment by nom** [comment]
Thanks! One more q: Did you train with guidance scale 3.5 or 1?

**Comment by ykj467422034** [comment]
> Hi, great work on the first working inpainting model for Flux!你好，Flux 的第一个修复模型做得很棒！
> 
> I'm working on reproducing the training to make sure I can finetune the model. Right now, my finetuning seems to destroy the inpainting after only 100 steps of training. I'm wondering what it could be.我正在努力重现训练，以确保我可以微调模型。现在，我的微调似乎只经过 100 步训练就破坏了修复。我想知道它可能是什么。
> 
> 1. One key hyperparameter seems to be the controlnet conditioning_scale. What did you set it to for training? The default is 1.0, the docs recommend a range of 0.9-0.95 for inference and the docstring example says 0.6.一个关键的超参数似乎是 controlnetconditioning_scale。您将其设置为什么来训练？默认值为 1.0，文档建议推理范围为 0.9-0.95，文档字符串示例显示为 0.6。
> 2. Which parameters did you train and which did you freeze?您训练了哪些参数，冻结了哪些参数？
> 3. Any other gotchas for training?训练还有其他问题吗？
> 
> Thanks! 谢谢！

Hello, may I refer to your training code? I am also researching the fine-tuning of FluxControlnetInpainting. Thanks！ @nom
